arrays/Kadane's Algorithm.md

```diff
@@ -0,0 +1,51 @@
+---
+id: kadanes-algorithm
+title: Kadane's Algorithm
+sidebar_label: Introduction to Kadane's Algorithm
```

Comment:
replace `sidebar_label: Introduction to Kadane's Algorithm` with `sidebar_label: "Introduction to Kadane's Algorithm"`